lib/targets/cuda/malloc.cpp

```diff
@@ -58,6 +60,13 @@ namespace quda
     }
   };
 
+#ifdef NVSHMEM_COMMS
+  // FIXME: Temporary value, based on DGX-1V layout
```

Comment:
Any updates?

Reply:
might be no longer necessary (see other comment)

Reply:
fixed.